Source/Scene/ModelAnimationCollection.js

```diff
@@ -450,11 +451,14 @@ ModelAnimationCollection.prototype.update = function (frameState) {
         delta = floor % 2 === 1.0 ? 1.0 - fract : fract;
       }
 
+      let startDelta = scheduledAnimation.startOffset;
       if (scheduledAnimation.reverse) {
         delta = 1.0 - delta;
+        startDelta = -startDelta;
       }
 
-      let localAnimationTime = delta * duration * scheduledAnimation.multiplier;
+      let localAnimationTime =
+        delta * duration * scheduledAnimation.multiplier + startDelta;
```

Comment:
This seems like the wrong place to be adding startDelta.

What happens when `loop` is set to `REPEAT` for example? Or worse, `MIRRORED_REPEAT`.

It also seems to be in the wrong units. If its supposed to be a "Fractional offset in to the animation's timeline" shouldn't it be multiplied by duration? By doing it this way, the user has to know what the duration is.

I think the best way to fix this is to add startDelta to delta on line 415. Then startDelta really can be a fraction, and REPEAT, MIRRORED_REPEAT will be properly handled, and play will be correctly handled when repeat is false...